basic-integration/surfly-integration-with-tag-manager.md

```diff
@@ -87,88 +85,90 @@ The tag is structured as follows:
     font-size: 13px;
   }
 
-  #btn-end-session {
+  #btn-end-session, #btn-start-session {
     display: none;
   }
 </style>
 
 <script>
-//Make sure that GTM doesn't hide the buttons
+  // Make sure that GTM does not hide the buttons
   document.getElementById('footer').parentElement.removeAttribute('style');
-  
-//Add the Surfly snippet
+
+  // Add the Surfly snippet -->
   (function(s,u,r,f,l,y){s[f]=s[f]||{init:function(){s[f].q=arguments}};
   l=u.createElement(r);y=u.getElementsByTagName(r)[0];l.async=1;
   l.src='https://surfly.com/surfly.js';y.parentNode.insertBefore(l,y);})
   (window,document,'script','Surfly');
 
-// Implement the Surfly init code, ADD YOUR OWN WIDGET KEY 
-  Surfly.init({widget_key:'** add your own Surfly widget key'}, function(init) {
+  // Implement the Surfly init code, ADD YOUR OWN WIDGET KEY
+  Surfly.init({widget_key:'** add your own Surfly widget key **'}, function(init) {
     if (init.success) {
-      if (Surfly.currentSession) {
-        // Laat binnen de sessie de btn-end-session zien
+      if (Surfly.isInsideSession) {
+        // Inside the session, display the btn-end-session
         document.getElementById('btn-end-session').style.display="block";
-        // Zorg ervoor dat de btn-start-session ruimte maakt om de session-id te laten zien
-        document.getElementById("btn-start-session").style.visibility="hidden";
+      }
```

Comment:
A remark about codestyle. Usually they write like

```javascript
if (...) {
  ...
} else {
  ...
}
```

or

```javascript
if (...)
{
  ...
}
else
{
  ...
}
```

in JS, the former is more popular, and that's what we are using at Surfly. Note the whitespaces around condition parentheses.

It so happened, we are mostly following the [Google style guide](https://google.github.io/styleguide/jsguide.html). It is quite verbose, but describes everything, so you can use it as a reference. Things like whitespaces around binary operators (`=`, `+`, etc) are easy to remember and get used to. A simple lifehack to fix the code style, is to use [jsbeautifier](http://jsbeautifier.org/). Its default settings are nearly identical to the code style we are using. Just make sure you are pasting a valid HTML or valid JS.

If you are using Sublime, [SublimeLinter](http://sublimelinter.readthedocs.io/en/latest/installation.html) package with [SublimeLinter-eslint](https://github.com/roadhump/SublimeLinter-eslint) does a good job (it checks the codestyle on every file save)